src/views/Registration/RegistrationDefineBusiness.vue

```diff
@@ -9,7 +9,18 @@
       <v-card flat class="mt-5">
         <NameRequestInfo />
 
-        <!-- FUTURE: Business Type goes here -->
+        <template v-if="isTypePartnership">
+          <v-divider class="mx-6" />
+
+          <!-- Business Type -->
+          <header id="business-type-header" />
+          <BusinessTypeConfirm
+            class="py-8 px-6"
+            :class="{ 'invalid-section': getShowErrors && !businessTypeConfirmValid }"
+            @update:businessTypeConfirm="setRegistrationBusinessTypeConfirm($event)"
+            @valid="onBusinessTypeConfirmValidEvent($event)"
+          />
```

Comment:
Don't forget to pass in the initial value (as restored from a draft filing).

Reply:
Thanks. I added the initial value to both files: BusinessNumber.vue and RegistrationDefineBusiness.vue.

Reply:
Yup, looks good 👍